internal/providers/terraform/azure/app_service_certificate_order.go

```diff
@@ -26,6 +26,7 @@ func NewAzureRMAppServiceCertificateOrder(d *schema.ResourceData, u *schema.Usag
 	if d.Get("product_type").Type != gjson.Null {
 		productType = d.Get("product_type").String()
 	}
+	productType = strings.ToLower(productType)
```

Comment:
The cost components use case-insensitive filter, this is used for breakdown output only.